algoliasearch-common/src/main/java/com/algolia/search/objects/SynonymQuery.java

```diff
@@ -25,12 +29,21 @@ public SynonymQuery setQuery(String query) {
     return this;
   }
 
-  public List<String> getTypes() {
-    return types;
+  public String getAnchoring() {
```

Comment:
Why is there anchoring on `SynonymQuery`? That notion exists in the rule, but not in the synonyms AFAIK.